waspc/README.md

```diff
@@ -362,7 +362,12 @@ NOTE: If building of your commit is suddenly taking much longer time, it might b
 If it happens just once every so it is probably nothing to worry about. If it happens consistently, we should look into it.
 
 ### Typical Release Process
-- Ensure that all starter templates in `starter` repo are working with the version of Wasp we are about to release and upgrade their version of Wasp to the new one.
+- Starter templates
+  - Context: they are used by used by `wasp new`, you can find reference to them in `Wasp.Cli. ... .StarterTemplates`.
```

Comment:
`Wasp.Cli. ... .StarterTemplates` is a placeholder you wanted to replace with a proper module path or are you keeping vague intentionally?

Reply:
I wanted to keep it vague intentionally, to make it a bit more robust to changes in the future. I could make it more concrete if needed.